apiary.apib

```diff
@@ -677,7 +674,7 @@ A new item can be put in the Queue, this item will auto expire after 30 seconds.
 
         {
             "url": "http://www.example.com/",
-            "id": "fuSHr0sRQ1usugvheahwQ"
+            "id": "fasdasdasdasdasdasdas"
```

Comment:
Hmm... This is public documentation, it would be nice to have something more realistic here